tests/identifiers/fixtures/datacite_post_metadata_response.xml

```diff
@@ -1,5 +1,5 @@
 <resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://datacite.org/schema/kernel-4" xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.dat/metadata.xsd">
-  <identifier identifierType="DOI">10.70102/FK2osf.io/yvzp4</identifier>
+  <identifier identifierType="DOI">{}</identifier>
```

Comment:
Am I reading this correctly that we're writing the string literal '{}' here so that we can `format` into it in mock_datacite above?

In any case, embedding that level of Python knowledge into an xml file should *at least* be documented, and I'd encourage to format with a keyword to make it at least a little bit clearer.

Ideally, we would be parsing/modifying the xml in code to be more explicit, but I get that that may require external libraries that we don't want to depend on if we're rarely using XML.

Reply:
Yeah, I rewrote this now with more explicit valider XML